src/utils/plot-templates/dark.js

```diff
@@ -0,0 +1,226 @@
+const dark = {
```

Comment:
Could you add a description of this file and where the format comes from in a comment block at the top? 
Even if it's just a link to Plotly docs would help I think.